neural_compressor/adaptor/torch_utils/smooth_quant.py

```diff
@@ -57,7 +57,10 @@ def move_input_to_device(input, device=torch.device('cpu')):
 
 ##TODO potential bug, data type
 def forward_wrapper(model, input, device=torch.device('cpu')):
-    input = move_input_to_device(input, device)
+    try:
+        input = move_input_to_device(input, device)
```

Comment:
@yintong-lu  could you help support recursive solution

Reply:
yes, I will create a seperate pr for it.